webrender/src/tiling.rs

```diff
@@ -818,6 +825,28 @@ enum AlphaBatchKind {
     BoxShadow,
 }
 
+bitflags! {
+    pub flags AlphaBatchKeyFlags: u8 {
+        const NEEDS_CLIPPING  = 0b00000001,
+        const AXIS_ALIGNED    = 0b00000010,
+        const COMPLEX         = 0b00000100,
```

Comment:
The COMPLEX flag is set there, but it's never actually tested against - the fn transform_kind() function just checks the valule of AXIS_ALIGNED (since the presence of one implies the absence of the other). Something seems to have gotten confused with the merge / rebase too - it looks like there is part of another commit in this PR now?